app/views/get_a_teacher_relocation_payment/claims/subject.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for(
+  :page_title,
+  page_title(
+        t("get_a_teacher_relocation_payment.forms.subject.question.#{@form.answers.application_route}"),
+    journey: current_journey_routing_name,
+    show_error: @form.errors.any?
+  )
+) %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for(
+      @form,
+      url: claim_path(current_journey_routing_name),
+      builder: GOVUKDesignSystemFormBuilder::FormBuilder
+    ) do |f| %>
+      <% if f.object.errors.any? %>
+        <%= render("shared/error_summary", instance: f.object) %>
+      <% end %>
```

Comment:
```suggestion
      <%= f.govuk_error_summary %>
```
now that we're actually using the GOVUK formbuilder are we able to move away from our homebrew version?

Reply:
Yeah that sounds better. There's other IRP forms that are still using the old approach so I've opened [this ticket](https://dfedigital.atlassian.net.mcas.ms/jira/software/projects/CAPT/boards/132?assignee=712020%3A565ea239-10c3-40ce-a2e0-3a46fcb91702&selectedIssue=CAPT-1735) to tackle migrating them all together so I don't miss any